.gitignore

```diff
@@ -22,4 +22,7 @@
 /config/master.key
 
 # Ignore Coverage
-/coverage
\ No newline at end of file
+/coverage
+
+# Ignore Spotify credentials
+/config/initializers/spotify_setup.rb
```

Comment:
That is not how you hide credentials from version control. Please use the encrypted credentials method in Rails and reinstate this file to VCS

Reply:
This one is fixed now!